### PR TITLE
fix(add-project-summary): Ui fixes in add project summary page

### PR DESCRIPTION
**Pull request by amritkv**
UI fixes in add project summary page

### CODE REVIEW

File: src/app/[locale]/projects/add/page.tsx [hunk 1]
```diff
@@ -7,211 +7,456 @@
...
         <>
             <SearchUsersModalComponent />
-            <div className="mx-5 mt-1">
-                <div className="row mt-2">
-                    <div className="col col-sm-3">
-                        <div className="list-group">
-                            <Link href="/projects/add/summary">Summary</Link>
-                            <Link href="/projects/add/administration">Administration</Link>
-                            <Link href="/projects/add/linkedProjects">Linked Releases and Projects</Link>
+            <div className='mx-5 mt-1'>
```
Comment (hoangnt2) [inline src/app/[locale]/projects/add/page.tsx:26]: You can use SideBar Component for tab switcher. Refer to Component Detail page
https://github.com/eclipse-sw360/sw360-frontend/blob/main/src/app/%5Blocale%5D/components/detail/%5Bid%5D/components/DetailOverview.tsx

File: src/components/AddAdditionalRoles.tsx [hunk 2]
```diff
@@ -60,17 +65,26 @@ export default function AddAdditionalRolesComponent() {
...
+                                           onChange={e => handleInputChange(e, j)}
+                                           className="form-control"
+                                           placeholder={`Enter email`}
+                                           aria-describedby={`Email`} />
                                 </div>
                                 <div className="col-lg-2">
-                                    <button type="button" onClick={() => handleRemoveClick(j)} className={`fw-bold btn btn-light ${styles['button-plain']}`}><i className="bi bi-trash3-fill"></i></button>
+                                    <button type="button" onClick={() => handleRemoveClick(j)}
+                                            className={`fw-bold btn btn-light ${styles['button-plain']}`}>
+                                                <Image src={deleteIcon} alt='Delete Icon'/>
```
Comment (hoangnt2) [inline src/components/AddAdditionalRoles.tsx:77]: use Component FaTrashAlt  from react-icons
```
import { FaTrashAlt } from 'react-icons/fa'
```

Comment (amritkv) [reply src/components/AddAdditionalRoles.tsx]: changes are incorporated

**Review by heliocastro — APPROVED** [review]
Good to go

**Pull request merged**